plugin/src/test/groovy/org/gradle/testretry/testframework/TestNGConfigCacheFuncTest.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.testretry.testframework
+
+import org.gradle.testretry.AbstractConfigCacheFuncTest
+import spock.lang.Ignore
+import spock.lang.Issue
+
+@Ignore
```

Comment:
I couldn't figure out how to use that in this situation because `PendingFeature` is only applicable to methods, not classes.  Since the methods are (by design) in the parent class, that wouldn't work.

Another option would be to build a subclass-specific check into the `shouldTest` so that we automatically start checking this after 6.7 (when my fix should be available).  I'll do that unless you have an idea how to make `PendingFeature` work.

Reply:
Agree with your proposed solution 👍